app/src/main/java/inspire2connect/inspire2connect/home/selectMiscActivity.java

```diff
@@ -48,27 +48,27 @@ protected void onCreate(Bundle savedInstanceState) {
 
     @Override
     public void onClick(View view) {
+//        if (view == misc_buttons[0]) {
+//            Intent i = new Intent(selectMiscActivity.this, urlActivity.class);
+//            i.putExtra("url", getString(R.string.covid_map));
+//            i.putExtra("name", getString(R.string.india_covid_map_tile));
+//            startActivity(i);
+//        }
         if (view == misc_buttons[0]) {
```

Comment:
Change this to switch case for view.getId. Will be easier to add more options and you wont have to comment stuff either for removal